TechliftTelegramBot/Services/BotCommandCheckService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TechliftTelegramBot.Models;
+using Telegram.Bot;
+using Telegram.Bot.Types;
+
+namespace TechliftTelegramBot.Services
+{
+    public class BotCommandCheckService : IBotCommandCheckService
+    {
+        private readonly ILogger _logger;
+        private readonly ApplicationConfiguration _config;
+        private readonly TelegramBotClient _botclient;
+        private IEnumerable<BotCommand> CommandsToBeSet;
```

Comment:
naming is not consistent for private variables

Reply:
Also why do we need to declare this private variable? its only needed when we call the GetCommandsAsync so why to have class level properties?